File: 1024hw/hw12.c
#include <stdio.h>
#include <windows.h>

int main() {
	int principal,year; float interest,result;
	principal = 0; interest = 0.0; year = 0;
	printf("Enter the principal : ");
	scanf_s("%d",&principal);
	printf("Enter the rate of interest : ");
	scanf_s("%f",&interest);
	printf("Enter the number of the years : ");
	scanf_s("%d",&year);
	result = principal + (principal * interest * year / 100);
	printf("After %d years at %.2f%%, the invest will be worth $%.2f",year,interest,result);
	system("pause");
}

File: 1108/hw12.py
principal = float(input("Enter the principal : "))
rate = float(input("Enter the interest of the rate : "))
year = int(input("Enter the number of years : "))
print("After",year,"years at",rate,"%, the 1nvestment will be $",round(principal*(1+(year*rate/100)),3))